cluster/gce/manifests/network-proxy.manifest

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Can you make this somewhat more formatted? Why not use yaml?

Reply:
Consistency with the other manifests I am used to working with. Will look at switching to yaml on my next pass.

Reply:
Switched.